theano/tensor/signal/tests/test_downsample.py

```diff
@@ -193,12 +193,21 @@ def test_DownsampleFactorMax(self):
 
                 # DownsampleFactorMax op
                 maxpool_op = DownsampleFactorMax(maxpoolshp,
-                                                 ignore_border=ignore_border,
-                                                 mode=mode)(images)
+                                                ignore_border=ignore_border,
+                                                mode=mode)(images)
```

Comment:
You changed the alignment of this line and the one above. This break the pep8 compliance and this checked by a test. So just undo the change on those 2 lines to fix one test failure.